bin/analyze

```diff
@@ -0,0 +1,367 @@
+                        default=None,
+                        help="GitHub API token")
+
+    parser.add_argument("-r", "--repo",
+                        required=True,
+                        help="GitHub repository, as 'owner/repo'")
+
+    parser.add_argument("-s", "--since",
+                        default=None,
+                        help="Start date for item consideration. ('%%Y-%%m-%%d' format)")
+
+    parser.add_argument("-u", "--until",
+                        default=None,
+                        help="End date for item consideration. ('%%Y-%%m-%%d' format)")
+
+    parser.add_argument("-cat", "--categories",
+                        default=[COMMIT_CATEGORY],
+                        nargs='+',
+                        choices=[COMMIT_CATEGORY, ISSUE_CATEGORY, PULL_REQUEST_CATEGORY],
+                        help="Possible options: %(choices)s (any combination)")
+
+    parser.add_argument("-c", "--conds",
+                        default=[],
+                        nargs='+',
+                        choices=['MergeExclude', 'EmptyExclude', 'MasterInclude'],
+                        help="Restrictions on which commits to include."
+                        " Possible options: %(choices)s (any combinations)")
+
+    parser.add_argument("-i", "--is-code",
+                        default=['Naive'],
+                        nargs='+',
+                        choices=['Naive', 'PostfixExclude', 'DirExclude'],
+                        help="Definition of Source Code."
+                             "Possible options: %(choices)s (any combination)")
+
+    parser.add_argument("-pf", "--postfixes-to-exclude",
+                        default=['.md', 'README'],
+                        nargs='+',
+                        help="Files to be excluded based on their extension."
+                        "Examples: .md, README")
+
+    parser.add_argument("-de", "--dirs-to-exclude",
+                        default=['tests', 'bin'],
+                        nargs='+',
+                        help="Files to be excluded based on their path."
+                             "Examples: tests, bin, docs")
+
+    parser.add_argument("-p", "--period",
+                        default='M',
+                        help="period for time-series: 'M', 'W', 'D', etc."
+                             "Any valid pandas Period frequency")
+
+    parser.add_argument("-o", "--output-formats",
+                        default=['json'],
+                        nargs='+',
+                        choices=['json', 'pdf', 'images'],
+                        help="Possible options: %(choices)s (any combination)")
+
+    parser.add_argument("-d", "--debug",
+                        action='store_true')
+
+    parser.add_argument("-w", "--write-to",
+                        default='results_dir',
+                        help="Path for output results")
+    return parser.parse_args()
+
+
+def run_metrics(items, categories, date_range, is_code, conds, period):
```

Comment:
missing `:param` for categories

Reply:
`period` is not used in this function, why?

Reply:
Sorry, that shouldn't be there.